GsaGH/Parameters/1_Properties/GsaProp2d.cs

```diff
@@ -4,289 +4,235 @@
 using System.Globalization;
 using System.Linq;
 using GsaAPI;
+using GsaGH.Helpers.GsaAPI;
 using OasysUnits;
 using OasysUnits.Units;
-using OasysGH;
-using GsaGH.Helpers.GsaAPI;
-using System.Collections.ObjectModel;
-using Grasshopper.Kernel.Types;
 using Rhino.Geometry;
 
-namespace GsaGH.Parameters
-{
+namespace GsaGH.Parameters {
   /// <summary>
   /// Prop2d class, this class defines the basic properties and methods for any <see cref="GsaAPI.Prop2D"/>
   /// </summary>
-  public class GsaProp2d
-  {
+  public class GsaProp2d {
     #region fields
-    private int _id = 0;
+    private int _id;
     private Guid _guid = Guid.NewGuid();
     private GsaMaterial _material = new GsaMaterial();
     private Prop2D _prop2d = new Prop2D();
     private Plane _localAxis = Plane.Unset;
     #endregion
 
     #region properties
-    internal Prop2D API_Prop2d
-    {
-      get
-      {
-        return this._prop2d;
+    internal Prop2D ApiProp2d {
```

Comment:
internal properties should be _camelCase?

Reply:
but what when we already have private field?

Reply:
yea, good point.. Did a quick google search but didn´t get an answer.
But I also think we should try and stick to the google style guide in this case somehow!

Reply:
can we leave it for now? and do it in other ticket then?

Reply:
![image](https://user-images.githubusercontent.com/83698580/228807640-bbc688fe-a170-4343-b11e-167c9a58a877.png)
everything inside class is private by default, and in this example (from google style guide) prop is in PascalCase